### PR TITLE
Woche8

### PR DESCRIPTION
**Pull request by Koch-Dustin**
Changes:
   - Add trivia
   - Add homepage

PR: #10

### CODE REVIEW

**Review by stefanluth — COMMENTED** [review]
Why is week 0 - 3 included and deleted?

**Comment by Koch-Dustin** [comment]
> 

I don't know why it was there i just deleted it because i don't need it in this Branch

**Comment by stefanluth** [comment]
Merging this branch will delete the files on develop. Undo the deletion.

**Comment by Koch-Dustin** [comment]
> Merging this branch will delete the files on develop. Undo the deletion.

I don't know how to Undo it, could you help me when you have time for it?

**Comment by stefanluth** [comment]
Try reverting the commit which deleted the files:

```zsh
git revert d893744
git add .
git commit .
git push
```

File: homepage/Kontakt/kontakt.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <br>
+                <span>Weiteres: Über Socialmedia</span>
+            </div>
+
+            <br>
+        </div>
+
+    </body>
+
+</html>
```
Comment (stefanluth) [inline homepage/Kontakt/kontakt.html:67]: EOF

File: homepage/Portfolio/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+menu1_is_open = false;
+menu2_is_open = false;
+menu3_is_open = false;
+
+function open_on_click(number) {
+    num = number;
+    if(menu1_is_open) {
```
Comment (stefanluth) [inline homepage/Portfolio/index.js:8]:
```suggestion
    if (menu1_is_open) {
```
Everywhere

**Review by stefanluth — CHANGES_REQUESTED** [review]
1. Add EOFs
2. Use 2 spaces for indentation in HTML, JS, CSS files
3. Add proper spacings after if, else, etc.

File: homepage/Kontakt/kontakt.html (hunk 1 above)
Comment (FrederikFricke) [inline homepage/Kontakt/kontakt.html:67]: End of file

File: homepage/Portfolio/portfolio.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                <img src="img/BeispielBild.jpeg" alt="">
+                <br>
+                <p>Projekt 3 ist ein Projekt wo es darum geht (Beispiel 1) und (Beispiel 2) das besondere hier dran ist das (Beispiel 3)</p>
+            </div>
+
+            <br>
+        </div>
+    </body>
+
+</html>
```
Comment (FrederikFricke) [inline homepage/Portfolio/portfolio.html:60]: end of file

File: homepage/Portfolio/style.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+.content_container h2 {
+    color: white;
+    margin-top: 2vh;
+}
+
+.content_container p {
+    color: white;
+    font-size: 1.2em;
+}
```
Comment (FrederikFricke) [inline homepage/Portfolio/style.css:148]: end of file

File: homepage/Socialmedia/Socialmedia.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                    <a href="https://www.snapchat.com/de-DE"><i class='bx bxl-snapchat' ></i></a>
+                    <a href="https://de-de.facebook.com/"><i class='bx bxl-facebook-square' ></i></a>
+                </li>
+            </ul>
+
+            <br>
+        </div>
+    </body>
+
+</html>
```
Comment (FrederikFricke) [inline homepage/Socialmedia/Socialmedia.html:41]: end of file

File: homepage/Socialmedia/style.css [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+.content_container ul li a i {
+    font-size: 5em;
+    color: white;
+}
+
+.content_container h2 {
+    color: white;
+    margin-top: 2vh;
+}
```
Comment (FrederikFricke) [inline homepage/Socialmedia/style.css:87]: end of file

File: trivia/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+            }
+
+            function change_display_part_one(display) {
+                const AnswerAlert = document.getElementById("CorrectAnswer");
+
+                if(IncorrectPartOne == false) {
+                    change_content_part_one("Correct");
+                    AnswerAlert.style.color = "green";
+                }else {
+                    change_content_part_one("Incorrect");
```
Comment (FrederikFricke) [inline trivia/index.html:24]:
```suggestion
                if (IncorrectPartOne == false) {
                    change_content_part_one("Correct");
                    AnswerAlert.style.color = "green";
                } else {
                    change_content_part_one("Incorrect");
```

Check other places as well

File: trivia/index.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+            }
+
+            function answer_on_click(isCorrect) {
+                IncorrectPartOne = true;
+                if (isCorrect) IncorrectPartOne = false;
+                change_display_part_one("block")
+            }
+
+            // TODO Part 2
+
```
Comment (FrederikFricke) [inline trivia/index.html:44]: Todo? still needed?

**Review by FrederikFricke — CHANGES_REQUESTED** [review]

File: homepage/Portfolio/index.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    }
+
+    if(menu3_is_open) {
+        if(number == 3) {
+            document.getElementById("projekt_three_arrow").src = "img/Arrow_Right.png";
+            document.getElementById("projekt_three_container").style.display = "none";
+            menu3_is_open = false;
+        }
+    }else {
```
Comment (Nicojele) [inline homepage/Portfolio/index.js:42]: add space

File: homepage/Portfolio/index.js [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+function open_on_click(number) {
+    num = number;
+    if(menu1_is_open) {
+        if(number == 1) {
+            document.getElementById("projekt_one_arrow").src = "img/Arrow_Right.png";
+            document.getElementById("projekt_one_container").style.display = "none";
+            menu1_is_open = false;
+        }
+    }else {
```
Comment (Nicojele) [inline homepage/Portfolio/index.js:14]: same

File: homepage/Portfolio/index.js [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    }
+
+    if(menu2_is_open) {
+        if(number == 2) {
+            document.getElementById("projekt_two_arrow").src = "img/Arrow_Right.png";
+            document.getElementById("projekt_two_container").style.display = "none";
+            menu2_is_open = false;
+        }
+    }else {
```
Comment (Nicojele) [inline homepage/Portfolio/index.js:28]: same

File: trivia/styles.css [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+    line-height: 1.8;
+    width: 25%;
+}
+
+input[type="text"]:hover {
+    background-color: #f5f5f5;
+    transition: color 2s ease-in-out, background-color 0.15s ease-in-out;
+}
+
+.section h4{
```
Comment (Nicojele) [inline trivia/styles.css:64]: same

**Review by Nicojele — CHANGES_REQUESTED** [review]

File: trivia/index.html [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+          Facebook
+        </button>
+        <button id="ButtonFour" onclick="answer_on_click(true)">tumblr</button>
+      </div>
+
+      <div class="section">
+        <h2>Part 2: Free Response</h2>
+        <hr />
+        <!-- TODO: Add free response question here -->
+        <h4 id="IncorrectAnswerTwo">Incorrect</h4>
```
Comment (FrederikFricke) [inline trivia/index.html:111]: Todo still needed?

**Review by FrederikFricke — CHANGES_REQUESTED** [review]

File: homepage/Kontakt/style.css [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+    text-align: end;
+    margin-left: auto;
+    user-select: none;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    border-bottom: 1px solid white;
+}
+
+.hyperlinkbar nav{
```
Comment (Nicojele) [inline homepage/Kontakt/style.css:31]: add space

File: homepage/Startseite/styles.css [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+    text-align: end;
+    margin-left: auto;
+    user-select: none;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    border-bottom: 1px solid white;
+}
+
+.hyperlinkbar nav{
```
Comment (Nicojele) [inline homepage/Startseite/styles.css:36]: add Space

**Review by FrederikFricke — APPROVED** [review]

**Review by Nicojele — CHANGES_REQUESTED** [review]

**Review by Nicojele — APPROVED** [review]
Add BEM

File: homepage/Portfolio/index.js [hunk 16]
```diff
@@ -1,50 +1,49 @@
...
-    } else {
-        if (number == 1) {
-            document.getElementById("projekt_one_arrow").src = "img/Arrow_Down.png";
-            document.getElementById("projekt_one_container").style.display = "block";
-            menu1_is_open = true;
-        }
+  num = number;
+  if (menu1_is_open) {
+    if (number == 1) {
+      document.getElementById("projekt_one_arrow").src = "img/Arrow_Right.png";
```
Comment (stefanluth) [inline homepage/Portfolio/index.js:9]:
```suggestion
  const var = document.getElementById("projekt_one_arrow");
  if (menu1_is_open) {
    if (number == 1) {
      var.src = "img/Arrow_Right.png";
```

You can extract the elements into variables.

File: homepage/Kontakt/style.css [hunk 17]
```diff
@@ -0,0 +1,84 @@
...
+
+.header {
+  width: 100%;
+  height: 10vh;
+  background-color: rgb(36, 36, 36);
+  display: flex;
+  justify-content: center;
+}
+
+.hyperlinkbar {
```
Comment (DevBastiS) [inline homepage/Kontakt/style.css:15]:
```suggestion
.hyper-link-bar {
```

File: homepage/Kontakt/style.css [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+  height: 50px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.hyperlinkbar a:hover {
+  background-color: rgb(59, 59, 59);
+}
+
+.content_container {
```
Comment (DevBastiS) [inline homepage/Kontakt/style.css:54]:
```suggestion
.content-container {
```

**Review by DevBastiS — CHANGES_REQUESTED** [review]

**Review by stefanluth — APPROVED** [review]

**Pull request merged**